openfecli/commands/atommapping.py

```diff
@@ -69,7 +69,7 @@ def generate_mapping(mapper, mol1, mol2):
 def atommapping_print_dict_main(mapper, mol1, mol2):
     """Main function for generating and printing out the mapping"""
     mapping = generate_mapping(mapper, mol1, mol2)
```

Comment:
Shouldn't here and above also change to molA and molB so everything else makes more sense?

Reply:
yeah probably, these got missed by the rename hammer, I guess because they're variable names?